2200004_김장현/단어.py

```diff
@@ -0,0 +1,54 @@
+t = int(input())
+
+for i in range(1, t+1):
+    n, k = map(int, input().split())
+    puzzle = []
+
+    for j in range(n):
+        puzzle.append(list(map(int, input().split())))
+        
+    word = []
+    cnt = 0
+    for a in range(n):
+        for b in range(n):
+            if puzzle[a][b] == 0:
+                if cnt not in [0, 1]:
+                    word.append(cnt)
+                    cnt = 0
+                else:
+                    cnt = 0
```

Comment:
이중 리스트 탐색을 정말 잘해주셨네요! 👍 
```suggestion
    cnt = 0
    ans = 0
    for a in range(n):
        for b in range(n):
            if puzzle[a][b] == 0:
                if cnt == k:
                    ans += 1
                cnt = 0
```
빈칸의 길이를 잘 찾아주셨으니, 이제 답을 저장할 `ans` 변수를 만들고 `k`개의 빈 칸이 나올 때마다 답을 1 씩 늘려주시면 됩니다 😄 
다른 부분에도 위와 같은 느낌으로 적용해주신 뒤, `ans`를 출력해주시면 정답이에요! 👍